### PR TITLE
tar: add Turkish translation

### PR DESCRIPTION
**Pull request by seppacar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.tr/common/tar.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# tar
+
+> Arşivleme aracı.
+> Genellikle gzip veya bzip2 gibi bir sıkıştırma yöntemiyle birleştirilir.
+> Daha fazla bilgi için: <https://www.gnu.org/software/tar>.
```
Comment (kbdharun) [inline pages.tr/common/tar.md:5]:
```suggestion
> Daha fazla bilgi: <https://www.gnu.org/software/tar>.
```
I don't think we need for (için) here in order to better align with our other pages. [Reference](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py)

File: pages.tr/common/tar.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Sıkıştırılmış bir arşiv dosyasını hedef dizine çıkarın:
+
+`tar xf {{source.tar[.gz|.bz2|.xz]}} --directory={{directory}}`
+
+- Sıkıştırılmış bir arşiv oluşturun ve sıkıştırma yöntemini seçmek için arşiv sonekini kullanın:
+
+`tar caf {{target.tar.xz}} {{file1}} {{file2}} {{file3}}`
+
+- Bir tar arşivinin içeriğini listeler:
```
Comment (kbdharun) [inline pages.tr/common/tar.md:31]: Mention `verbosely`'s Turkish equivalent word here.

File: pages.tr/common/tar.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+`tar cf {{target.tar}} {{file1}} {{file2}} {{file3}}`
+
+- Bir gzip arşivi oluşturun ve dosyaya yazın:
+
+`tar czf {{target.tar.gz}} {{file1}} {{file2}} {{file3}}`
+
+- Göreceli yolları kullanarak bir gzip arşivi oluşturun:
+
+`tar czf {{target.tar.gz}} --directory={{path/to/directory}} .`
```
Comment (kbdharun) [inline pages.tr/common/tar.md:17]:
```suggestion
`tar czf {{hedef.tar.gz}} --directory={{dizin/yolu}} .`
```

File: pages.tr/common/tar.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- Sıkıştırılmış bir arşiv dosyasını geçerli dizine ayrıntılı şekilde çıkarın:
+
+`tar xvf {{source.tar[.gz|.bz2|.xz]}}`
+
+- Sıkıştırılmış bir arşiv dosyasını hedef dizine çıkarın:
+
+`tar xf {{source.tar[.gz|.bz2|.xz]}} --directory={{directory}}`
```
Comment (kbdharun) [inline pages.tr/common/tar.md:25]:
```suggestion
`tar xf {{kaynak.tar[.gz|.bz2|.xz]}} --directory={{dizin}}`
```
 Feel free to replace `kaynak` with some other work if necessary.

File: pages.tr/common/tar.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+- Sıkıştırılmış bir arşiv oluşturun ve sıkıştırma yöntemini seçmek için arşiv sonekini kullanın:
+
+`tar caf {{target.tar.xz}} {{file1}} {{file2}} {{file3}}`
```
Comment (kbdharun) [inline pages.tr/common/tar.md:29]:
```suggestion
`tar caf {{hedef.tar.xz}} {{dosya1}} {{dosya2}} {{dosya3}}`
```
Feel free to replace `hedef` with some other work if necessary.

File: pages.tr/common/tar.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
+# tar
+
+> Arşivleme aracı.
+> Genellikle gzip veya bzip2 gibi bir sıkıştırma yöntemiyle birleştirilir.
+> Daha fazla bilgi için: <https://www.gnu.org/software/tar>.
+
+- Bir arşiv oluşturun ve dosyaya yazın:
+
+`tar cf {{target.tar}} {{file1}} {{file2}} {{file3}}`
```
Comment (kbdharun) [inline pages.tr/common/tar.md:9]:
```suggestion
`tar cf {{hedef.tar}} {{dosya1}} {{dosya2}} {{dosya3}}`
```

File: pages.tr/common/tar.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+- Bir gzip arşivi oluşturun ve dosyaya yazın:
+
+`tar czf {{target.tar.gz}} {{file1}} {{file2}} {{file3}}`
```
Comment (kbdharun) [inline pages.tr/common/tar.md:13]:
```suggestion
`tar czf {{hedef.tar.gz}} {{dosya1}} {{dosya2}} {{dosya3}}`
```

File: pages.tr/common/tar.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+- Göreceli yolları kullanarak bir gzip arşivi oluşturun:
+
+`tar czf {{target.tar.gz}} --directory={{path/to/directory}} .`
+
+- Sıkıştırılmış bir arşiv dosyasını geçerli dizine ayrıntılı şekilde çıkarın:
+
+`tar xvf {{source.tar[.gz|.bz2|.xz]}}`
```
Comment (kbdharun) [inline pages.tr/common/tar.md:21]:
```suggestion
`tar xvf {{kaynak.tar[.gz|.bz2|.xz]}}`
```

File: pages.tr/common/tar.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+`tar xf {{source.tar[.gz|.bz2|.xz]}} --directory={{directory}}`
+
+- Sıkıştırılmış bir arşiv oluşturun ve sıkıştırma yöntemini seçmek için arşiv sonekini kullanın:
+
+`tar caf {{target.tar.xz}} {{file1}} {{file2}} {{file3}}`
+
+- Bir tar arşivinin içeriğini listeler:
+
+`tar tvf {{source.tar}}`
```
Comment (kbdharun) [inline pages.tr/common/tar.md:33]:
```suggestion
`tar tvf {{kaynak.tar}}`
```

File: pages.tr/common/tar.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+- Şablonla eşleşen dosyaları arşivden çıkarın:
+
+`tar xf {{source.tar}} --wildcards "{{*.html}}"`
```
Comment (kbdharun) [inline pages.tr/common/tar.md:37]:
```suggestion
`tar xf {{kaynak.tar}} --wildcards "{{*.html}}"`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and Welcome to tldr. I have some suggestions for this page.

In future, **Translate command arguments inside `{{}}`** as it will be useful for native readers. I have made suggestions for the current PR. You can refer to [common-arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) if necessary.

Tip: Create a single commit by batching all my suggestion in the `Files changed` tab of this PR.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**